backend/app/services/ml_engine.py

```
@@ -0,0 +1,432 @@
+    docs_path = paths[0]
+    try:
+        kb = get_kb_orchestrator().get_kb(docs_path, use_bm25=True)
+        if not kb.get("chunks"):
+            warnings.append("После разбиения документов не получено ни одного чанка")
+        return {
+            **state,
+            "chunks": kb.get("chunks", []),
+            "warnings": warnings,
+            "errors": errors,
+        }
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)